app/assets/stylesheets/styles/components/guides/components/_actions_modal.css.scss

```diff
@@ -0,0 +1,33 @@
+.actions-modal {
+  -webkit-transition: opacity 200ms ease-in;
+  -moz-transition: opacity 200ms ease-in;
+  transition: opacity 200ms ease-in;
+
+  .actions ul li{
```

Comment:
Selector should have depth of applicability no greater than 3, but was 4<br>Opening curly brace `{` should be preceded by one space